internal/core/metadata/v2/application/device.go

```diff
@@ -100,3 +100,33 @@ func AllDeviceByServiceName(offset int, limit int, name string, ctx context.Cont
 	}
 	return devices, nil
 }
+
+// CheckDeviceExistsById checks the device existence by id
+func CheckDeviceExistsById(id string, dic *di.Container) (exists bool, edgeXerr errors.EdgeX) {
+	if id == "" {
+		return exists, errors.NewCommonEdgeX(errors.KindContractInvalid, "id is empty", nil)
+	}
+	_, err := uuid.Parse(id)
+	if err != nil {
+		return exists, errors.NewCommonEdgeX(errors.KindInvalidId, "fail to parse id as an UUID", err)
+	}
+	dbClient := v2MetadataContainer.DBClientFrom(dic.Get)
+	exists, edgeXerr = dbClient.CheckDeviceExistsById(id)
+	if edgeXerr != nil {
+		return exists, errors.NewCommonEdgeXWrapper(err)
+	}
+	return exists, nil
+}
+
+// CheckDeviceExistsByName checks the device existence by name
+func CheckDeviceExistsByName(name string, dic *di.Container) (exists bool, err errors.EdgeX) {
```

Comment:
rename all the function name to `DeviceNameExists`

Reply:
Renamed DeviceIdExists and DeviceNameExists function.